test/OpenTelemetry.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs

```diff
@@ -37,6 +39,133 @@ public void RuntimeMetricsAreCaptured()
             meterProvider.ForceFlush(MaxTimeToAllowForFlush);
             Assert.True(exportedItems.Count > 1);
             Assert.StartsWith(MetricPrefix, exportedItems[0].Name);
+
+            var gcCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.gc.collections.count");
+            var sumReceived = GetLongSum(gcCountMetric);
+            Assert.True(sumReceived >= 0);
+
+#if NETCOREAPP3_1_OR_GREATER
+            var gcAllocationSizeMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.gc.allocations.size");
+            Assert.True(GetLongSum(gcAllocationSizeMetric) > 0);
+#endif
+
+#if NET6_0_OR_GREATER
+            // Supposedly to pass if no garbage collection occurred before. However one occurred, which is out of control of the code.
+            // Assert.False(exportedItems.Exists(i => i.Name == "process.runtime.dotnet.gc.committed_memory.size"));
+            // Assert.False(exportedItems.Exists(i => i.Name == "process.runtime.dotnet.gc.heap.size"));
+#endif
+
+            var assembliesCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.assemblies.count");
+            Assert.True(GetLongSum(assembliesCountMetric) > 0);
+        }
+
+#if NET6_0_OR_GREATER
+        [Fact]
+        public void RuntimeMetrics_GcAvailableAfterFirst()
```

Comment:
nit: the class is already RuntimeMetrics - we can simpify the name to GcMetricsTest()/similar